test/PrintAsObjC/extensions.swift

```diff
@@ -95,8 +118,7 @@ extension GenericClass {
 class NotObjC {}
 extension NotObjC {}
 
-// CHECK-LABEL: @interface NSObject (SWIFT_EXTENSION(extensions))
-// CHECK-NEXT: @end
+// NEGATIVE-NOT: @interface NSObject (SWIFT_EXTENSION(extensions))
 // NEGATIVE-NOT: @interface NSObject{{$}}
 // NEGATIVE-NOT: @class NSObject
 extension NSObject {}
```

Comment:
Makes sense. In `master` it looks like the test first asserts that there is an extension, then asserts that there is no class defined. Due to the sorted nature of this header would it be more correct to first assert that the class declaration is not there followed by the extension check?